frontend/amundsen_application/static/js/pages/TableDetailPage/ReportTableIssue/styles.scss

```diff
@@ -13,16 +14,16 @@ $submit-row-gap: 15%;
   display: block;
   height: auto;
   left: 25px;
-  min-height: 300px;
-  padding: 32px;
+  min-height: $modal-size;
```

Comment:
Did you notice this wasn't the same number?

Reply:
I did, I thought you meant when you said "the same as min-height" to just make them the same since it was a minimum value and didn't end up changing the height. But I can go ahead and change it to be their own values

Reply:
Ah, sorry for the confusion!
If they looks the same, go ahead with this!

Reply:
Sounds good, I will leave it then! The modal looks like it is always taller than the width with the contents it has anyway